stl/inc/algorithm

```diff
@@ -3567,6 +3567,77 @@ _FwdIt2 unique_copy(_ExPo&&, _FwdIt1 _First, _FwdIt1 _Last, _FwdIt2 _Dest) noexc
 }
 #endif // _HAS_CXX17
 
+#ifdef __cpp_lib_concepts
+namespace ranges {
+    // VARIABLE ranges::reverse
+    // clang-format off XXX
+    // concept-constrained for strict enforcement as it is used by several algorithms
+    template <bidirectional_iterator _It>
+    // requires permutable<_It>

```

Comment:
Why is this commented out?

Reply:
Pilot error (see details in earlier comment).